fix: Update prose and screenshots in light of profiles (#259)
citynetwork/docs: docs/howto/openstack/nova/new-server.md

file first](../../getting-started/enable-openstack-cli.md).

=== "{{gui}}"
    On the top right-hand side of the {{gui}}, click the _Create_ button. A
    new pane titled _Create_ will slide into view from the right-hand side

>     of the browser window.

Author: This must have happened due to re-applying the max width policy and then the content tab indenting.

    ![Create new object](assets/new-server/shot-01.png)

    You will notice several rounded boxes on that pane, each for defining,
    configuring, and instantiating a different {{company}} Cloud object. Go
    ahead and click the _Server_ box. A new pane titled _Create a Server_
    will slide over. At the top, type in a name for the new server and

>     select one of the available regions.

Author: Ack.

    ![Create new server](assets/new-server/shot-02.png)

>     Below the region selection is a drop-down menu where you must select a
>     _server profile_. Currently, four profiles are available: _Generic_,
>     _High CPU_, _Low Latency Disk_, and _GPU_. Selecting each of those gives
>     you access to a subset of all available
>     [_flavors_](../../../reference/flavors/index.md). Now, as the profile
>     names suggest, each profile but the _Generic_ points to a set of flavors
>     suitable for servers intended to run specific types of applications.
>     More specifically, the _Generic_ profile is for general-purpose cloud
>     servers, the _CPU_ profile is for servers designed to host CPU-intensive
>     applications, the _Low Latency Disk_ is for servers that boot off of a
>     local low-latency disk, whereas the _GPU_ profile is for servers that
>     boot off of a local SSD disk and, most importantly, have direct access
>     to an NVIDIA vGPU. We should point out that _not_ all profiles may be
>     available in any region. Let us choose the _Generic_ profile for now,
>     which, by the way, is available in any of the regions.

Reviewer: This wall of text is difficult to read when rendered. For better legibility, I would suggest to use bullet points for enumerating the server profiles. Also, use the imperative mood, and edit out fluff like "by the way".

```suggestion
    Below the region selection is a drop-down menu where you must select a
    _server profile_. Currently, four profiles are available:
    
    *  _Generic_,
    * _High CPU_,
    * _Low Latency Disk_, and
    *  _GPU_.
    
    Selecting one of those gives
    you access to a subset of all available
    [_flavors_](../../../reference/flavors/index.md).
    
    Now, as the profile
    names suggest, each profile but the _Generic_ points to a set of flavors
    suitable for servers intended to run specific types of applications:
    
    * The _Generic_ profile is for general-purpose cloud
    servers.
    * The _CPU_ profile is for servers designed to host CPU-intensive
    applications.
    * The _Low Latency Disk_ profile is for servers that boot off of a
    local low-latency disk.
    * The _GPU_ profile is for servers that
    boot off of a local SSD disk and, most importantly, have direct access
    to an NVIDIA vGPU. 
    
    We should point out that _not_ all profiles may be
    available in any region. Choose the _Generic_ profile for now,
    which, is available in any of the regions.
```

    > Please keep in mind that profile selection is not available via the
    > OpenStack CLI. When creating servers using the `openstack` client,
    > though, you can still select flavors suitable for specific types of

>     > workloads.

Reviewer: I don't think this belongs in the GUI content tab.

Author: My thinking is that whenever you read about creating servers using the OpenStack CLI, this comment doesn't make sense: server profiles are not a thing, at least not as far as the OpenStack CLI is concerned. On the other hand, server profiles can make sense only as far as specific dashboards (like ours) are concerned.

Reviewer: Well but it doesn't make sense in the GUI tab either. If you don't want to move it to the CLI tab, then I think you can just remove this paragraph.

Author: It would look even more out-of-place/unnatural/weird in the CLI tab, so I am removing the paragraph.

Reviewer: Ack, works for me!

    available OS images to boot the new server off of. To pick one of the
    images, click on the dropdown menu on the right. For this how-to guide,
    we have chosen _ubuntu_ in general and _Ubuntu 22.04 Jammy Jellyfish_ in
    particular.

    ![Boot source](assets/new-server/shot-04.png)

    Next up, notice that the _Boot Target_ is preselected for you; depending
    on the server profile you have already selected, the boot target will be
    either _Volume_ (_Generic_, _High CPU_, _GPU_ profiles) or _Ephemeral_
    (_Low Latency Disk_ profile). Regarding the server's CPU core count and
    amount of memory, set the
    [_Flavor_](../../../reference/flavors/index.md) accordingly. We suggest
    selecting a flavor that specifies a server with 1 CPU core and 1GiB of
    RAM. Please note that, depending on the chosen flavor, the estimated
    monthly cost of the server changes. (While a server is shut off you are
    still getting charged for it, but less so.) At any time, this estimated
    cost is displayed in the green rectangular area at the top. Something
    else that affects the cost is the size of the root device. Take a look

>     at the _Volume_ parameter below, and notice the default (in

Reviewer: Here, again, we have a long wall of text that needs breaking up.

Author: I'm breaking up (the wall of text, that is).

    and click the _Server_ box. A new pane titled _Create a Server_ will
    slide over. At the top, type in a name for the new server and select one
    of the available regions.

    ![Create new server](assets/new-server/shot-02.png)

    Below the region selection is a drop-down menu where you must select a
    _server profile_. Currently, four profiles are available:

    *  _Generic_,
    * _High CPU_,
    * _Low Latency Disk_, and
    *  _GPU_.

    Selecting one of those gives you access to a subset of all available
    [_flavors_](../../../reference/flavors/index.md).

    As the profile names suggest, each profile but the _Generic_ points to a
    set of flavors suitable for servers intended to run specific types of
    applications:

    * The _Generic_ profile is for general-purpose cloud servers.
    * The _CPU_ profile is for servers designed to host CPU-intensive
      applications.
    * The _Low Latency Disk_ profile is for servers that boot off of a local
      low-latency disk.
    * The _GPU_ profile is for servers that boot off of a local SSD disk
      and, most importantly, have direct access to an NVIDIA vGPU.

>     We should point out that _not_ all profiles may be available in any

Reviewer: Micro-nit: reading this out loud it seems odd that "not" is emphasised here. I think you have two options:

1. Emphasise "all" instead:
```suggestion
    We should point out that not _all_ profiles may be available in any
```
2. Drop the emphasis altogether:
```suggestion
    We should point out that not all profiles may be available in any
```

Author: It is odd indeed. And since some emphasis wouldn't hurt here, I am emphasizing "all".

    ```
## Connecting to the server console
=== "{{gui}}"
    While viewing information regarding your server, you may get its public
    IP  address (e.g., from the _Addresses_ tab) and connect to it remotely
    via SSH. Alternatively, you may launch a web console and log in; click
    on the three-dot icon on the right of the server header, and from the

>     pop-up menu that appears select _Remote Console_.

Reviewer: Again, the rewrap made it pretty difficult to pick out that the only change here is the addition of the two words `via SSH`.

Author: Understood. Again, that must have happened because of the maximum width and indenting for content tabs. (And yes, after reading the paragraph, I had to add those two words.)

    ![Launch remote console](assets/new-server/shot-10.png)

    A new window pops up, and that's your web console to your {{brand}}
    server. Please note that this window cannot be resized but can be opened

>     on a new browser window or tab.

Reviewer: No changes in this paragraph, it seems?

Author: Yes, no changes in this paragraph.

    ```
## Connecting to the server console
=== "{{gui}}"
    While viewing information regarding your server, you may get its public
    IP  address (e.g., from the _Addresses_ tab) and connect to it remotely
    via SSH. Alternatively, you may launch a web console and log in; click
    on the three-dot icon on the right of the server header, and from the
    pop-up menu that appears select _Remote Console_.

    ![Launch remote console](assets/new-server/shot-10.png)

    A new window pops up, and that's your web console to your {{brand}}
    server. Please note that this window cannot be resized but can be opened

>     on a new browser window or tab.

Reviewer: It just occurred to me that I typically open things _in_ a new browser window or tab, not _on_ one.

Author: Ja, ich auch :|
